Base/usr/share/man/man1/xargs.md

```diff
@@ -0,0 +1,33 @@
+## Name
+
+xargs - build and execute commandlines from input
+
+## Synopsis
+
+```**sh
+$ xargs [options...] command [initial-arguments...]
+```
+
+## Description
+
+`xargs` reads items from a stream, delimited by some blank character (`delimiter`), and executes the `command` as many times as there are items, with any processed `initial-arguments`, possibly followed by an item read from the input.
+Each argument in `initial-arguments` is processed by replacing any occurence of the `replacement string` with the input item, and treating the entire resulting value as _one_ argument.
+If no argument in `initial-arguments` contains the `replacement string`, an argument is added at the end of the list containing only the `replacement string`.
```

Comment:
I'm sorry but "replacement string" is not a valid identifier/token (as it contains the space), so it feels wrong to put it in monospace. Either add a dash (`replacement-string`), or better, just say `placeholder`.

Reply:
I swapped it for placeholder along with the first suggestion, no worries :)